staging/cse/windows/kubeletfunc.ps1

```diff
@@ -220,11 +220,12 @@ function Add-KubeletNodeLabel {
     $labelList = $KubeletNodeLabels -split ","
     foreach ($existingLabel in $labelList) {
         if ($existingLabel -eq $Label) {
-            Write-Log "found existing kubelet node label $existingLabel, will continue without adding anything"
+            # we need to use Write-Host since Write-Log will add content to the return value, which we don't want
+            Write-Host("found existing kubelet node label $existingLabel, will continue without adding anything" | Timestamp)

```

Comment:
Please use a global variable instead of return in powershell scripts. Please also update Get-TagValue and Remove-KubeletNodeLabel too.
Please still use Write-Log instead of Write-Host

Reply:
https://github.com/Azure/AgentBaker/pull/5151